spec/features/breadcrumbs_spec.rb

```diff
@@ -8,6 +8,17 @@
   let(:hearing_id_from_fixture) { '345be88a-31cf-4a30-9de3-da98e973367e' }
   let(:hearing_id) { hearing_id_from_fixture }
 
+  before do
```

Comment:
This before and after aren't needed - you can remove them and the test still passes.